pkg/vfs/reader.go

```diff
@@ -224,7 +224,7 @@ func (s *sliceReader) run() {
 		_ = f.r.m.InvalidateChunkCache(meta.Background, inode, indx)
 		if f.tried > f.r.maxRetries {
 			s.done(err, 0)
-		} else {
+		} else { // Why retry here since we already have retried in `cachedStore.load`?
```

Comment:
This is another level of retry. The slices obtained from the `meta.Read` method may be changed.

Reply:
Got it, updated